### PR TITLE
Counterspell chennai

### PR DESCRIPTION
**Pull request by Vipremigini**
Ignore the first 3 commits

### CODE REVIEW

File: components/Chennai/Guilds.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        </Tilt>
+      </div>
+
+      <div className="flex justify-center leading-[3rem] tracking-wider mx-6">
+        <div className="max-w-6xl px-6 py-6 mt-6 border-4 border-dashed border-pink">
+          <p className="text-3xl neuebit">
+            {/* TODO: Change `Example City` to the name of your city */}
+            You can join the guild you identify with for exclusive swag and
+            badges! You also get to compete against other guilds in friendly
+            activities/challenges throughout Counterspell Example City!
```
Comment (samdev-7) [inline components/Chennai/Guilds.js:50]:
```suggestion
            activities/challenges throughout Counterspell Chennai!
```

File: components/Chennai/Hero.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          <div className="hidden absolute top-[90%] lg:top-[145%] left-[9%] w-[80%] lg:scale-[130%] bg-[#090808] h-screen" />
+
+          <div className="flex items-center justify-center text-center">
+            <div className="z-50 absolute top-[43%] lg:top-[47%] text-[60%] sm:text-[80%] md:text-[100%] lg:text-[140%] tracking-wide">
+              <p className="text-[#CFD8ED] retro">
+                A BEGINNER FRIENDLY GAME JAM
+              </p>
+              <p className="text-pink retro text-[80%] sm:text-[90%] lg:text-[90%] mt-2">
+                {/* TODO: Replace `EXAMPLE CITY` with your city */}
+                NOVEMBER 23-24, 2024 • EXAMPLE CITY
```
Comment (samdev-7) [inline components/Chennai/Hero.js:82]:
```suggestion
                NOVEMBER 23-24, 2024 • CHENNAI
```

File: pages/Chennai.js
Comment (samdev-7) [inline pages/Chennai.js:1]: You'll need to change the filename of this to be lowercase, so chennai.js.

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost there! A few more changes.

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/Chennai**](https://counterspell-h7314rzfr.hackclub.dev/Chennai) |
|:-:|
|<a href="https://counterspell-h7314rzfr.hackclub.dev/Chennai">Screenshot of /Chennai</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/oXPzqi4mlOwGOmX4aI4shySRLNJUCTCYAOzkXi4PM83AVCHQTlEtx4uF8vBwJTbBcG1jEwIfD61EKd8UjwhweA.png">(view full size)</a> |







Commit d526a8055f84bc76ee0e6c94429421ccf55c31b2 (https://counterspell-h7314rzfr.hackclub.dev).

**Comment by Vipremigini** [comment]
Made the requested changes

**Comment by Vipremigini** [comment]
thank you

**Pull request merged**